src/elements/content-sidebar/activity-feed/Avatar.js

```diff
@@ -14,46 +14,46 @@ type Props = {
 
 type State = {
     avatarUrl?: ?string,
+    isPending: boolean,
 };
 
+// Note: setting a key is required, re-render with a new user is not implemented
```

Comment:
Which key are you referring to? I don't see one in this component.

Reply:
It needs a key when used, because it doesn't have a componentDidUpdate to handle refetching when the user identity changes `<Avatar key={user.id} />`

Reply:
gotcha, thanks